src/sign-in-blazorserver/README.md

```diff
@@ -0,0 +1,112 @@
+<!-- Keeping yaml frontmatter commented out for now
+---
+# Metadata required by https://docs.microsoft.com/samples/browse/
+# Metadata properties: https://review.docs.microsoft.com/help/contribute/samples/process/onboarding?branch=main#add-metadata-to-readme
+languages:
+- csharp
+page_type: sample
+name: "Blazor Server application app that makes a request to the Graph API after signing in the user"
+description: "This ASP.NET Core 6 Blazor Server application app signs in the user and then makes a request to Microsoft Graph for the user's profile data."
+products:
+- azure
+- azure-active-directory
+- ms-graph
+urlFragment: ms-identity-docs-code-app-csharp-blazorserver
+---
+-->
+
+# ASP.NET Core | Blazor Server application | user sign-in, protected web API access (Microsoft Graph) | Microsoft identity platform
+
+<!-- Build badges here
+![Build passing.](https://img.shields.io/badge/build-passing-brightgreen.svg) ![Code coverage.](https://img.shields.io/badge/coverage-100%25-brightgreen.svg) ![License.](https://img.shields.io/badge/license-MIT-green.svg)
+-->
+
+This ASP.NET Core Blazor Server application application authenticates a user and then makes a request to the Graph API as the authenticated user. The response to the request is presented to the user.
+
+![A screenshot of a Blazor Server application application displaying a response from Microsoft Graph.](./app-signed-in.png)
+
+## Prerequisites
+
+- Azure Active Directory (Azure AD) tenant and the permissions or role required for managing app registrations in the tenant.
+- [.NET 6.0 SDK](https://dotnet.microsoft.com/download/dotnet/6.0)
```

Comment:
Does this need the ASP.NET Core 6 SDK instead?  (I'm not fully up to date with what SDKs ship what now)

Reply:
this SDK will provider the users with the ASP.NET Core runtime and the scaffoldings/project-templates for `web*`. 

Alternatively, if we wanted to streamline this a bit more, we could just mention the runtime instead. It wouldn't provide a full fleshed dev experience through.

Reply:
Perfect, good to know.  Nothing to change then.